pkg/resource/deploy/deployment.go

```diff
@@ -68,8 +68,12 @@ type Options struct {
 	DestroyProgram bool
 	// if specified, only operate on the specified resources.
 	Targets UrnTargets
+	// if specified, ignore the specified resources
+	Excludes UrnTargets
```

Comment:
Same here, probably worth keeping targets and excludes together rather than interspersing it.